packages/ui/src/machines/authenticator/index.ts

```diff
@@ -42,26 +42,25 @@ export function createAuthenticatorMachine() {
           invoke: [
             {
               // TODO Wait for Auth to be configured
```

Comment:
Oh yeah, good question. Yes, ideally we would run this after we know `Amplify.configure` has been completed.  This has been the case because we `Amplifty.configure` are usually placed at app entry point, but we have no semantic guarantee over that 😛

Reply:
edited for wording *